client/src/app/site/users/components/user-list/user-list.component.html

```diff
@@ -12,12 +21,30 @@
     <!-- prefix column -->
     <ng-container matColumnDef="group">
         <mat-header-cell *matHeaderCellDef mat-sort-header> Group </mat-header-cell>
-        <mat-cell *matCellDef="let user"> {{user.groups}} {{user.structureLevel}} </mat-cell>
+        <mat-cell *matCellDef="let user">
+            <div class='groupsCell'>
+                <span *ngIf="user.groups">
+                    <fa-icon icon="users"></fa-icon>
+                    {{user.groups}}
```

Comment:
Does angular automatically concat the group names to `group1,group2`? If so please make this custom. Add a whitespace after each comma and display max. 3 groups. If there are move append `, ...`.